content/docs/1.3.0/snapshots-and-backups/csi-snapshot-support/csi-volume-snapshot-associated-with-longhorn-snapshot.md

```diff
@@ -0,0 +1,90 @@
+---
+title: CSI VolumeSnapshot Associated with Longhorn Snapshot
+weight: 3
```

Comment:
Can we change this to weight 2?
I'd prefer to describe snapshot first then backup.

Reply:
Thank you. Fixed